### PR TITLE
The portfolio website

### PR DESCRIPTION
**Pull request by tanveerisonline**
Description Of project 👍 
1. Building mobile-first UI.
2. Creating a logo section.
3. Creating a hamburger icon for mobile first UI.
4. Creating a bio section.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+  <title>Portfolio</title>
+</head>
+<body>
+  <main class="container">
+  
+    <header class="header">
+      <ul>
+        <li><img src="./assets/Time.png" alt="clock" class="top"></li>
+        <li><img src="./assets/Group 3.png" alt="bars" class="top"></li>
+      </ul>
```
Comment (zieeco) [inline index.html]: - Hey, please kindly remove this part as they are not part of the design as the design was been viewed on a mobile device, that's you could see in the design.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+    </header>
+    <nav>
+      <a href="#" target="_parent" class="logo">My Portfolio</a>
+      <img src="./assets/Union.png" class="mobile_menu_icon"/>
+    </nav>
```
Comment (zieeco) [inline index.html]: - Please, let's move the nav into the header after you've removed the part I requested you remove from the previous comment.
- Also, let's use the correct font, color, etc to improve our design.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+
+    <section>
+      <div class="hero">
+        <p class="author">I’m Tanveer Glad to see you!</p>
```
Comment (zieeco) [inline index.html]: - Please, let's use the h1 tag here as this is the main heading of your portfolio.

File: index.html [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+           <a href="#" class="connect_button">let's connect</a>
+      </div>      
+    </section>
+    <aside>
+      <ul>
+        <li><img src="./assets/Vector.svg" alt="twitter"></li>
+        <li><img src="./assets/Linkedin icon.svg" alt="Linkedin"></li>
+        <li><img src="./assets/Vector-2.svg" alt="m-icon"></li>
+        <li><img src="./assets/Vector-1.svg" alt="github"></li>
+        <li><img src="./assets/Vector-3.svg" alt="victory"></li>
```
Comment (zieeco) [inline index.html]: - Let's wrap each image in an anchor tag as those should link to your social media platforms.

File: styles.css [hunk 5]
```diff
@@ -0,0 +1,82 @@
...
+  padding: 20px 10px 8px 10px;
+  color: #344563;
+}
+
+.connect_button {
+  text-transform: uppercase;
+  color: #7f8cff;
+  font-size: 20px;
+  padding: 0 10px 0 10px;
+}
```
Comment (zieeco) [inline styles.css]: - Please, let's improve our design by using the correct font family, font-weight, font-size, color, etc here to meet the design as much as we can below:

*design template*

Screenshot from 2022-10-28 01-19-15

*your design*

Screenshot from 2022-10-28 01-27-17

File: README.md [hunk 6]
```diff
@@ -0,0 +1,97 @@
...
+
+## Have you already completed the Sass lessons?
+
+If you completed the Sass lessons you may prefer to work with SCSS files, if that's the case you need to perform
+a few modifications to the linter configuration files.
+
+1. In the file `.github/workflows/linters.yml` [line 48](https://github.com/microverseinc/linters-config/blob/master/html-css/.github/workflows/linters.yml#L48) replace `"**/*.{css,scss}"` with `"**/*.scss"`
+2. Use `npx stylelint "**/*.scss"` to lint you SCSS files and not the generated CSS
+
+Once you've done the previous steps you can focus on your SCSS files and not in fixing the generated CSS output
```
Comment (zieeco) [inline README.md]: - Hey, Please, let's remove all these lines in the readme and update it using the microverse readme [template](https://github.com/microverseinc/readme-template) and we should customize it to suit the needs our the current project to make the readme more professional.

**Review by zieeco — CHANGES_REQUESTED** [review]
Hi @tanveerisonline ,


Great job so far! 👏 
You've done well, however, there are some issues that you still need to work on to go to the next project, but you are almost there!


## To highlight 🎯

- No linter errors.✔️
- Nice design. ✔️
- Descriptive PR. ✔️


## Required Changes ♻️
_Check the comments under the review._





## Optional suggestions
- N/A

**_Every comment with the **[OPTIONAL]** prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._**


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by tanveerisonline** [comment]
Do I have to remove the border that I have used for the whole design or
should I keep it as it is now?

File: index.html [hunk 7]
```diff
@@ -0,0 +1,38 @@
...
+           don’t hestiate to contact me.</p>        
+           <a href="#" class="connect_button">let's connect</a>
+
+        <div class="social">
+        <a href="#"><img src="./assets/Vector.svg" alt="twitter"></a>
+        <a href="#"><img src="./assets/Linkedin icon.svg" alt="Linkedin"></a>
+        <a href="#"><img src="./assets/Vector-2.svg" alt="m-icon"></a>
+        <a href="#"><img src="./assets/Vector-1.svg" alt="github"></a>
+        <a href="#"><img src="./assets/Vector-3.svg" alt="victory"></a>
+        </div>
```
Comment (ha-manel) [inline index.html]: - [ ] I think you misunderstood the previous reviewer, the way you made this part as a `ul` and `li`s was correct 👍 but the `img` should be wrapped in an anchor tag. The final structure will look like this: 

```
<ul>
   <li> <a href="website/yourHandle"><img....></a> </li>
</ul>
```

**Review by ha-manel — CHANGES_REQUESTED** [review]
# STATUS: REQUIRED CHANGES ♻️

## Hi @tanveerisonline👋 
Great job so far, There are some changes that need to be implemented in order to proceed to the next project 👍  



## Highlights:  
* All tests passed ✔️
* Meaningful commit messages ✔️ 
* Professional README file ✔️

## Required Changes:
_Kindly check the comment below_

## Optional:
- [ ] It's better if your page would be full screen width (`100vw`) instead of fixed `375px`, that way your website would look awesome on any mobile screen 💯 

------
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 7 above)
Comment (tanveerisonline) [reply index.html]: The purpose of these anchor tags is to take users to the given social platforms so these links do exactly as they should so you cannot say that I have to make changes and use my last review for this exactly not. I am sorry but I think I used the correct way as you can check the tons of well-designed websites where best practices were used to build them and they same have done this as I did. you can check the Microsofts website.
thank and plz approve my project that will be your most kindness.

**Comment by medaminedev66** [comment]
Hello @tanveerisonline 👋,

_This is an additional review as a result of the second opinion request._

One of the project goals is how to use HTML semantics perfectly. A semantic element clearly describes its meaning to both the browser and the developer rather than merely defining its presentation or look.

Why does Semantic HTML matter? Writing semantic HTML makes your code easier to understand, making the source code more readable for other developers and browsers.

Since the social icons are unordered lists, we usually use `ul` tag to wrap all the social icons instead of `div` tag and use `li` tag to wrap each of the social icons for better HTML semantic usage.

I recommend implementing the changes requested by @ha-manel and requesting a new review.

Keep rocking 🚀!

**Review by BertrandConxy — APPROVED** [review]
Hi @tanveerisonline, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights 💪 
- No linter errors
- good PR description
- correct flow is used

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @BertrandConxy in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**